test/test.js

```diff
@@ -53,7 +46,7 @@ describe('Generate function', () => {
 
 describe('isOccurencesValid function returs false on obviously false numbers', () => {
```

Comment:
@UgurGumushan a couple of typos. Would you fix them in this PR? 
`returns` and the function name has double R -> `areOccurrencesValid` (should be changed everywhere).